src/components/routes/homepage/Homepage.js

```diff
@@ -1,22 +1,16 @@
 import React from 'react';
-import { useQuery } from '@apollo/react-hooks';
 import { Events } from './Events';
 import { Loading } from '../../shared/Loading';
-import { GET_ALL_EVENTS } from '../../../gql/events';
 import { EventFormContainer } from './EventFormContainer';
+import { useDataFetching } from '../../../custom-hooks/useDataFetching';
 
-export const Homepage = ({ me }) => {
-  const {
-    loading: eventsLoading,
-    error: eventsError,
-    data: eventsData
-  } = useQuery(GET_ALL_EVENTS);
+export const Homepage = () => {
+  const { data, loading, error } = useDataFetching();
 
-  if (eventsLoading) return <Loading />;
+  if (loading) return <Loading />;
+  if (error) console.log(error);
 
-  if (eventsError) console.log(eventsError);
-
-  const { events } = eventsData;
+  const { me, events } = data;
```

Comment:
Could you help removing `me` here, it's not actually being used inside `Events`. I did that because initially I thought deletion will be done on `Homepage`